wormhole-connect/src/icons/Routes/Wormhole.tsx

```diff
@@ -3,28 +3,15 @@ import React from 'react';
-        fill="white"
-      />
-      <path
-        d="M125.281 168.231C107.872 168.217 91.1781 161.299 78.8628 148.993C66.5475 136.687 59.616 119.999 59.5888 102.589C59.616 85.1798 66.5475 68.4918 78.8628 56.1861C91.1781 43.8804 107.872 36.962 125.281 36.9484C142.692 36.9586 159.387 43.8759 171.703 56.1823C184.019 68.4888 190.95 85.1788 190.973 102.589C190.946 119.999 184.015 136.687 171.699 148.993C159.384 161.299 142.691 168.217 125.281 168.231V168.231ZM125.281 41.2176C109.003 41.2277 93.393 47.6953 81.8776 59.2011C70.3622 70.7069 63.8816 86.311 63.8578 102.589C63.885 118.866 70.3673 134.467 81.8823 145.97C93.3974 157.473 109.005 163.938 125.281 163.949C141.557 163.938 157.165 157.473 168.68 145.97C180.195 134.467 186.677 118.866 186.705 102.589C186.681 86.311 180.2 70.7069 168.684 59.2011C157.169 47.6953 141.56 41.2277 125.281 41.2176V41.2176Z"
-        fill="white"
-      />
-      <path
-        d="M137.924 152.422C125.061 152.412 112.726 147.301 103.627 138.208C94.5276 129.116 89.4062 116.786 89.3858 103.922C89.4028 91.0568 94.5229 78.7233 103.623 69.6283C112.722 60.5333 125.059 55.4198 137.924 55.4096C150.79 55.4198 163.126 60.5333 172.226 69.6283C181.326 78.7233 186.446 91.0568 186.463 103.922C186.442 116.786 181.321 129.116 172.222 138.208C163.122 147.301 150.788 152.412 137.924 152.422V152.422ZM137.924 57.9737C125.739 57.9839 114.055 62.8272 105.436 71.4414C96.8169 80.0555 91.9669 91.7369 91.9499 103.922C91.9703 116.106 96.8217 127.784 105.44 136.395C114.059 145.006 125.741 149.848 137.924 149.858C150.108 149.848 161.789 145.006 170.408 136.395C179.027 127.784 183.878 116.106 183.898 103.922C183.881 91.7369 179.032 80.0555 170.413 71.4414C161.794 62.8272 150.11 57.9839 137.924 57.9737Z"
-        fill="white"
+        id="fullLogo"
+        d="m197.9535,174.6565c-7.318,12.735-20.858,20.566-35.532,20.566h-23.731v-80.03l-34.251,59.483c-7.3188,12.734-20.8583,20.565-35.5323,20.565h-23.5661v-118.1051h47.4067v79.8281l45.9427-79.7915v-.0366h47.389v80.1581l49.785-86.5071c2.013-3.513,1.976-7.8676-.201-11.2891C212.7015,23.2333,171.9905-.6805,125.7365.0148,55.203,1.1126-.3641,57.8872.0018,128.4215c.3659,70.222,57.4151,127.034,127.7107,127.034s127.711-57.178,127.711-127.711c0-11.363-1.5-22.377-4.299-32.8796-.842-3.1836-5.087-3.7691-6.734-.9331l-46.455,80.7067.018.018Z"
+        style={{ fill: '#fff', strokeWidth: '0px' }}
```

Comment:
this needs to be a variable color otherwise it's hard to see in light mode:
<img width="420" alt="image" src="https://github.com/wormhole-foundation/wormhole-connect/assets/897596/f2dbfb30-12c6-4c9b-9115-61a336c754e8">

Reply:
This should be resolved by now!